draft-irtf-cfrg-hpke.md

```diff
@@ -798,6 +817,21 @@ security models and assumptions, and do not consider attackers capable of quantu
 computation. A full proof of post-quantum security would need to take this
 difference into account, in addition to simply using a post-quantum KEM.
 
+## Domain Separation
+
+\[\[RFC editor: please change "RFCXXXX" to the correct number before
+publication.]]
+
+The KeySchedule procedure includes the domain separation string "RFCXXXX" in
+each Expand invocation. This ensures any secrets derived in HPKE are independent
+from those used in other protocols, even when derived from the same IKM (secret).
+This domain separation does not protect the KeySchedule 'secret' from use in other
+protocols.
+
+The KeySchedule procedure also separates each Hash invocation, used when deriving
+pskID and info hashes, with unique one-byte prefixes. This ensures that each Hash
+computation (or oracle query) is independent {{BDG20}}.
```

Comment:
I think you can remove "(or oracle query)".
I wouldn't use that reference either... There is nothing new here, that said I don't have a good one in mind : )

Reply:
I dropped the parenthetical. As I don't have a better reference for domain separation, I'm inclined to keep the citation. (Better to point somewhere than rely on folklore?)